mixpanel.go

```diff
@@ -61,7 +76,19 @@ func (m *Mixpanel) Engage(distinctId string, props Properties, op *Operation) er
 	props["mp_lib"] = library
 	props[op.Name] = op.Values
 
-	return m.makeRequestWithData("GET", "engage", props)
+	return m.makeRequestWithData("GET", "engage", props, sourceUser)
+}
+
+// UpdateProperties calls the engage endpoint, but doesn't set IP, city, country, on the profile.
+func (m *Mixpanel) UpdateProperties(distinctId string, props Properties, op *Operation) error {
```

Comment:
Would this be better named as `EngageAsScript` or something similar? `UpdateProperties` doesn't necessarily map to something in their API very clearly